.ci/ci_cache_kernel.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Source to trap error line number
+source "${script_dir}/../lib/common.bash"
+
+WORKSPACE=${WORKSPACE:-$(pwd)}
+kata_dir="/usr/share/kata-containers/"
+
+cache_build_kernel() {
+	kernel_path="${1}"
+	real_path=$(readlink -f ${kernel_path})
+	#Get version from binary (format: vmlinu{z|x}-4.19.24-25)
+	kernel_binary=$(basename ${real_path})
+	# Latest if the file to say what is the cached version
+	echo "${kernel_binary}" | cut -d- -f2- >  "latest"
+	cp "${real_path}" ./
+
+        sudo chown -R "${USER}:${USER}" ./
+	sha256sum "${kernel_binary}" >> "sha256sum-kernel"
+}
+
+mkdir -p "${WORKSPACE}/artifacts"
+pushd "${WORKSPACE}/artifacts"
+rm -f sha256sum-kernel
+cache_build_kernel "${kata_dir}/vmlinux.container"
+cache_build_kernel "${kata_dir}/vmlinuz.container"
+ls -la "${WORKSPACE}/artifacts/"
+popd
+sync
```

Comment:
why a `sync`?

Reply:
The job is running in a VM, this is going to be collected by the jenkins master node, do you want to be added as a comment ?

Reply:
Ah, I see. Yeah, if you are updating then add a note pls - thanks!

Reply:
done :)